pkg/quayio/quayio.go

```diff
@@ -0,0 +1,73 @@
+package quayio
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	// Sun, 01 Jul 18 00:00:00 +0000
+	lastModifiedTimestamp = 1530403200
```

Comment:
Can't we instead get the images from last two years or something instead of having fixed date here?

Reply:
IT is going to be configurable and by default, it will be e.g. 2h. So we will run it every hour and check repos which were modified during the last two hours. It has this value here for the initial retagging